CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+## Contributing to Gymkhana Central Website
+
+### Submitting a Pull Request (PR)
+
+1. Search GitHub for an open or closed PR that relates to your submission. You don't want to duplicate existing efforts.
+
+2. Be sure that an issue describes the problem you're fixing, or documents the design for the feature you'd like to add. 
+   Discussing the design upfront helps to ensure that we're ready to accept your work.
+
+3. Fork the Gymkhana-IITI repo.
+
+4. In your forked repository, make your changes in a new git branch:
+   - git checkout -b my-fix-branch master
+
+5. Commit your changes using a descriptive commit message that follows our commit message conventions. Adherence to these conventions is 
+necessary because release notes are automatically generated from these messages.
+   - git commit --all
+
+6. Push your branch to GitHub:
```

Comment:
Once go through these git command and write their simpler syntax that we use normally